addons/knowledge/views/knowledge_templates_frontend.xml

```diff
@@ -94,6 +94,7 @@
             <ul class="o_tree o_tree_workspace m-0 px-0 py-2">
                 <t t-call="knowledge.articles_template">
                     <t t-set="articles" t-value="visible_articles"/>
+                    <t t-set="portal_readonly_mode" t-value="True"/>
```

Comment:
This is already set to True here:
https://github.com/odoo/odoo/blob/master/addons/knowledge/controllers/main.py#L88